commcare_connect/opportunity/migrations/0055_payment_amount_usd.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 4.2.5 on 2024-10-03 17:40
+
+from django.db import migrations, models
+from django.db.models.functions import TruncDate
+from commcare_connect.opportunity.visit_import import get_exchange_rate
+
+def populate_amount_usd():
+    Payment = apps.get_model("opportunity.Payment")
+    payments = (
+        Payment.objects
+        .annotate(date_only=TruncDate('date_paid'))
+        .values('date_only', 'opportunity_access__opportunity__currency')
+        .distinct()
+    )
+
+    for payment in payments:
+        date_paid = payment['date_only']
+        currency = payment['opportunity_access__opportunity__currency']
+
+        if currency in ['USD', None]:
+            exchange_rate = 1
+        else:
+            exchange_rate = get_exchange_rate(currency, date_paid)
+        if not exchange_rate:
+            raise Exception(f"Invalid currency code {currency}")
```

Comment:
May be ignore the error and just print instead?

Reply:
I think this is fine. Everything should be valid at this point, and we know what the currencies are. I think one opportunity has a typo in the currency, that we can fix before you deploy, but it would be good to have a hard failure to force us to do so.